db/external_sst_file_ingestion_job.cc

```diff
@@ -557,6 +581,9 @@ Status ExternalSstFileIngestionJob::AssignGlobalSeqnoForIngestedFile(
       std::string seqno_val;
       PutFixed64(&seqno_val, seqno);
       status = rwfile->Write(file_to_ingest->global_seqno_offset, seqno_val);
+      if (status.ok()) {
+        status = SyncIngestedFile(rwfile.get());
```

Comment:
It seems to me we don't need to sync directory here, but I'm not very sure.

Reply:
Does this mean we may potentially have to sync a file twice if write_global_seqno is true?